gmsm/games-ios
# TabBar

**Goal**
Include `TabBar` in the project

**How was it done?**
I created a protocol to create cohesion between classes. After that, I created a class to implement the TabBar.

**How to test?**
When you run the application, you will see the screen divided into two tabs: TopGames and Favorites

**Evidence (optional)
![simulator screen shot - iphone se - 2018-08-09 at 11 25 46](https://user-images.githubusercontent.com/41014257/43905263-16d06e80-9bc7-11e8-8784-aa28ee5c89a9.png)
**
